app/lib/client_pdf_document.rb

```diff
@@ -2,7 +2,13 @@ class ClientPdfDocument
   def self.create_or_update(output_file:, document_type: , client:, filename:, tax_return: nil)
     tempfile = output_file
     tempfile.seek(0)
-    document = client.documents.find_or_initialize_by(document_type: document_type.key)
+    # TODO: after no document_types of "F13614C 2020" change back to => document = client.documents.find_or_initialize_by(document_type: document_type.key)
+    document =
+      if document_type == DocumentTypes::Form13614C && (client.documents.find_by(document_type: document_type.key) || client.documents.find_by(document_type: "F13614C 2020"))
```

Comment:
these `find_by`s could be simplified as such

`client.documents.find_by(document_type: [document_type.key, "F13614C 2020"])`

there might also be a craftier way to avoid the double-find but maybe it doesn't matter much because it's only temporary